sources/modules/VETravelPerformance/R/TravelTimeReliability.R

```diff
@@ -0,0 +1,352 @@
+# #===================
+# #TravelTimeReliability.R
+# #===================
+# 
+# #<doc>
+# #
+# ## TravelTimeReliability Module
+# #### October 01, 2020
+# #
+# # This module calculates travel time reliability measures for the model area. Average speeds at different congestion levels (None, Mod, Hvy, Sev, Ext) on freeways and arterials by marea calculated in the CalculateRoadPerformance  
+# # Module are used as inputs to calculate travel time index
+# # to calculate Travel Time Index 
+# # for Marea
+# ### Model Parameter Estimation
+# #
+# #This module has no estimated parameters.
+# #
+# ### How the Module Works
+# 
+# #
+# #This module calculates following metrics:
+# #
+# #* * Travel Time Reliability metric during moderate congestion on arterials for Marea
+# #* * Travel Time Reliability metric during heavy congestion on arterials for Marea
+# #* * Travel Time Reliability metric during severe congestion on arterials for Marea
+# #* * Travel Time Reliability metric during extreme congestion on arterials for Marea
+# #* * Travel Time Reliability metric during moderate congestion on freeways for Marea
+# #* * Travel Time Reliability metric during heavy congestion on freeways for Marea
+# #* * Travel Time Reliability metric during severe congestion on freeways for Marea
+# #* * Travel Time Reliability metric during extreme congestion on freeways for Marea
+# #
+# #</doc>
+# 
+# 
+# #=================================
+# #Packages used in code development
+# #=================================
+# #Uncomment following lines during code development. Recomment when done.
+# # library(visioneval)
+# # library(stringr)

```

Comment:
For testing, you could add magrittr and dplyr here. You need them explicitly marked as dependencies (see below where you import the pipe operator).